lifelines/fitters/mixture_cure_fitter.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import autograd.numpy as anp
+from autograd.scipy.special import expit
+from lifelines.fitters import ParametricUnivariateFitter
+
+# What should the name of the fitter actually be?
+class MixtureCureFitter(ParametricUnivariateFitter):
+    def __init__(self, base_fitter, *args, **kwargs):
+        self._base_fitter = base_fitter
+
+        # What should the variable that we use be called?
+        # Should raise an exception if that variable is in the base fitted parameter names
+
+        self._fitted_parameter_names = ["cured_"] + base_fitter._fitted_parameter_names
```

Comment:
Some suggestions:
`"logit_cured"`, `"logit_cured_prob"`.


Any reason to use the model the logit? The alternative is to model the probability and use bounds `(0, 1)` - this gives a much easier interpretation for end users.

Reply:
I have a data set for which I have built a mixture cure model as a ParametricRegressionFitter with several covariates. If I just use the identify link, then the fit does not converge. Instead I use logit and it works well. I assumed it would be the same here, but I'm actually unable to replicate using this univariate fitter. I've pushed a new commit for you to look at which models things as you've described.

Reply:
I've seen this too - for regression data, using unbounded parameterizations works much better. But for univariate fitters, it's less important. I think the code without link is better for this :+1: